### ISSUE TITLE
Chunks, Clear, Rate and other Bugs.

### ISSUE THREAD

**Issue by Maxetto** [IntellectualSites/PlotSquared#953]
After updating to latest commit, I think I've finally solved the /p auto issue... However /p clear and /p delete still destroy the world.

- [ ] /p clear and /p delete leave a hole in the plot (If you don't replicate this, try with more Plot Worlds or after some time. It doesn't happen straight after a start/restart)
Standing in this hole feels like gliding (like a chunk that didn't load client-side), yet if you check with WorldEdit it gives you AIR.
![2016-03-13_01 40 34](https://cloud.githubusercontent.com/assets/1360752/13726239/c477a760-e8be-11e5-83e3-dc22fa0239c3.png)

- [ ] Teleporting with Chunk Processor doesn't load chunks until you relog.

- [x] /p list returns "Possible values: [Ljava.lang.String;@7301fab" instead of "Possible values: shared, world, done (etc)"

- [x] /p rate next sometimes sticks to a plot that you already voted and won't let you go past it. Seems to happen with merged plots. I suggest, as a workaround, to create a new command to skip a vote "/p rate skip".

Revelant Info:
Using Commit [3fb6866](https://github.com/IntellectualSites/PlotSquared/commit/3fb686608badc57af17bf0b0e182fd650643cec5)
[Config](http://pastebin.com/T29euCLp)
Spigot 1.8.8 (git-Spigot-db6de12-18fbb24) - Offline Mode
AsyncWorldEdit v3.3.0
WorldEdit v6.1
2 PlotWorlds (32x and 64x) generated with Bukkit's Generator

**Comment by L4BORG** [comment]
Try redownloading the latest update? (silent update if you're downloading builds from spigotmc.org) I had exact the same issue and I think it's ok now... well, hopefully it won't come back in a few hours (last time I've checked /p clear was bugged in less then 30 min after server restart).

**Comment by Maxetto** [comment]
I'm building directly from Source. It's hard to be a build bug.
I have those kind of issues since I've updated to 3.3.X

**Comment by MattBDev** [comment]
Relevant info would mean you provide us a debug paste so we know all the information that is helpful to us

**Comment by MattBDev** [comment]
I think it also may be helpful if we included a githas him the version for people like you who compile the builds.

**Comment by Maxetto** [comment]
[HERE](http://pastebin.com/t3rGp8vz) it is. I redacted some private infos, like IPs. Hope you don't mind.

**Comment by Maxetto** [comment]
Chunks that doesn't regenerate also has a different Biome.
My Plots have FOREST Biome as default, but bugged chunks have PLAINS Biome.

**Comment by cicatrice74** [comment]
I have this issue too, after restart the p clear or p delete functions correctly but after some time the hole in the plot happens... anyone can fix this ?

**Comment by AipaMC** [comment]
I can also confirm this issue. Seems to work fine for a little while after a restart and then massive holes start forming when using /plot clear.

**Comment by L4BORG** [comment]
Yeah, 5 hours after restart /p clear is bugged again (on latest 3.3.1).

**Comment by MattBDev** [comment]
@Maxetto We don't have an official build server at the moment.

**Comment by L4BORG** [comment]
I can clear XX of plots, but only after restart... once /p clear is bugged... it would be bugged for everyone till next restat.

**Comment by Maxetto** [comment]
This is still not fully fixed. Using ChunkProcessor, even if I reset a plot and it's perfect, whenever I move around, the plot goes back to its bugged state...

**Comment by boy0001** [comment]
If it stops rendering when you move or look around then it's a client side rendering issue. The rate bug should going another ticket.
